spec/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Specification PR template
+When creating a specification project release review, create PRs with the content defined as follows.
+
+Include the following in the PR:
+- [ ] A directory in the form <project-name>/x.y where <project-name> is the specificaiton project name
+(micropfile-config, microprofile-health, ...) and x.y is the release major.minor version, and the directory contains the following.
+- [ ] Specification PDF in the form of microprofile-project-spec-x.y.pdf where
+_project_ is the microprofile specification short project name (config, health, ...)
+- [ ] Specification HTML in the form of microprofile-project-spec-x.y.html
+- [ ] A specification page named index.fmt in a TBD format.
+- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
+- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
+Instructions MAY be by reference.
+   - [ ] Updated release record
```

Comment:
What do you mean by updated release record?

Reply:
1, that it exists, and 2, that it has the correct date. The date determines when it can be scheduled for a review in the bi-weekly EMO release approval meetings.